### PR TITLE
Adding api method for updating cartridge group

### PR DESCRIPTION
**Pull request by Vishanth**
* Added API method for update cartridge group
* Updated wsdl

### CODE REVIEW

File: components/org.apache.stratos.autoscaler/src/main/java/org/apache/stratos/autoscaler/registry/RegistryManager.java [hunk 1]
```diff
@@ -348,6 +349,28 @@ public void persistServiceGroup(ServiceGroup servicegroup) {
...
+                throw new InvalidServiceGroupException(String.format("Cartridge group does not exist: " +
+                        "[cartridge-group] %s", serviceGroup.getName()));
+            }
+
+            persistServiceGroup(serviceGroup);
+
+            if (log.isDebugEnabled()) {
+                log.debug(String.format("Updated cartridge group: [group-name] %s", serviceGroup.getName()));
+            }
+        } catch (Exception e) {
```
Comment (lahirus) [inline components/org.apache.stratos.autoscaler/src/main/java/org/apache/stratos/autoscaler/registry/RegistryManager.java]: We can throw the exception to client.

Comment (imesh) [inline components/org.apache.stratos.autoscaler/src/main/java/org/apache/stratos/autoscaler/registry/RegistryManager.java]: Yes please throw an exception with a proper message.

Comment (Vishanth) [inline components/org.apache.stratos.autoscaler/src/main/java/org/apache/stratos/autoscaler/registry/RegistryManager.java]: Made the changes.

**Comment by Vishanth** [comment]
@imesh resolved the merge conflicts

**Pull request merged**